### ISSUE TITLE
Multi windows

### ISSUE THREAD

**Issue by gedw99** [goki/gide#301]
Sone users have multi monitor setups 

sone users like to dock windows etc 

it would be very possible to add this to the u deelyung framework so that Gide and others apps get this feature.

I have it working in gio on all OS’s and the code is simple

**Comment by kkoreilly** [comment]
Yep, in Cogent Code, if you click `Open` in a window where you already have a project open, it will create a new window with the other project, so you can have multiple windows easily (and we are working on improving multi-monitor support as you mentioned).
